pkg/httpmetrics/metrics.go

```diff
@@ -238,7 +238,10 @@ func tracerOptionsGCP(ctx context.Context) []trace.TracerProviderOption {
 	traceExporter, err := texporter.New(
 		// Avoid infinite recursion in trace uploads
 		//   https://github.com/open-telemetry/opentelemetry-go/issues/1928
-		texporter.WithTraceClientOptions([]option.ClientOption{option.WithTelemetryDisabled()}),
+		texporter.WithTraceClientOptions([]option.ClientOption{
+			option.WithTelemetryDisabled(),
+			option.WithUserAgent("otel-go"),
+		}),
```

Comment:
this adds it to the tracer exporter, which isnt the http metrics routed to gcp monitoring

i think you want to update the prometheus counter metric here
https://github.com/chainguard-dev/terraform-infra-common/blob/main/pkg/httpmetrics/metrics.go#L86-L92